dynamique/schema-dynamique.json

```
@@ -95,15 +95,17 @@
         },
         {
             "name": "horodatage",
-            "description": "TODO",
-            "example": "TODO",
+            "description": "Indique la date et heure de remontée de l’information publiée. Le format attendu est : iso 8601(aaaa-mm-jj 'T' hh:mm:ss:SSSZ)",
```

```suggestion
            "description": "Indique la date et heure de remontée de l’information publiée, formaté selon la norme ISO 8601",
```

Il existe plusieurs manières d'exprimer une date + heure au format ISO 8601. D'ailleurs dans l'exemple on  met une date time avec `+01:00`, et dans la description c'était avec `Z`. Il vaut mieux laisser les gens aller voir ce qu'est la norme et les variantes qu'elle offre, et mettre un exemple valide de notre côté je pense.